tests/playwright/src/utility/fixtures.ts

```diff
@@ -0,0 +1,61 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Page } from '@playwright/test';
+import { test as base } from '@playwright/test';
+
+import { WelcomePage } from '../model/pages/welcome-page';
+import { NavigationBar } from '../model/workbench/navigation';
+import { PodmanDesktopRunner } from '../runner/podman-desktop-runner';
+
+export type TestFixtures = {
+  pdRunner: PodmanDesktopRunner;
+  navigationBar: NavigationBar;
+  welcomePage: WelcomePage;
+  page: Page;
+};
+
+export type RunnerOptions = {
+  profile: string;
+  customFolder: string;
+  autoUpdate: boolean;
+  autoCheckUpdate: boolean;
+};
+
+export const test = base.extend<TestFixtures & RunnerOptions>({
+  profile: ['', { option: true }],
+  customFolder: ['podman-desktop', { option: true }],
+  autoUpdate: [true, { option: true }],
+  autoCheckUpdate: [true, { option: true }],
+  pdRunner: async ({ profile, customFolder, autoUpdate, autoCheckUpdate }, use) => {
```

Comment:
```suggestion
  runner: async ({ profile, customFolder, autoUpdate, autoCheckUpdate }, use) => {
```

Reply:
(can be follow-up)

Reply:
This will be part of the renaming of the PD runner class, for now we had a standard to reference the podman desktop runner object as pdRunner in all tests, so for now I'd rather keep it as is, but will rename the fixture to runner with the rest of the refactoring.

Reply:
yes I agree 👍